internal/chart/testdata/render_yamls/annotations-patch.yaml

```diff
@@ -2,5 +2,5 @@
   path: /metadata/annotations/app
   value: "app1"
 - op: add
-  path: /metadata/annotations/shipa.io~1is-shipa
+  path: /metadata/annotations/theketch.io~1is-shipa
```

Comment:
I think, it is `theketch.io` in main. @kavinaravind merged a fix recently.

https://github.com/theketchio/ketch/blob/main/internal/chart/testdata/render_yamls/annotations-patch.yaml#L5